release/rancher/rancher.go

```diff
@@ -53,28 +54,43 @@ const (
 	dockerService                 = "registry.docker.io"
 )
 
+var regsyncDefaultMediaTypes = []string{
+	"application/vnd.docker.distribution.manifest.v2+json",
+	"application/vnd.docker.distribution.manifest.list.v2+json",
+	"application/vnd.oci.image.manifest.v1+json",
+	"application/vnd.oci.image.index.v1+json",
+}
+
 var registriesInfo = map[string]registryInfo{
 	"registry.rancher.com": {
-		BaseURL: rancherRegistryBaseURL,
-		AuthURL: sccSUSEURL,
-		Service: sccSUSEService,
+		BaseURL:     rancherRegistryBaseURL,
+		AuthURL:     sccSUSEURL,
+		Service:     sccSUSEService,
+		UserEnv:     "{{env \"PRIME_REGISTRY_USERNAME\"}}",
```

Comment:
I think backticks are appropriate here and then you don't need to escape the double quote.